pages/photos.vue

```diff
@@ -33,7 +42,7 @@
 
 <script setup lang="ts">
 import { type Photo } from "~/types";
-import { useStorage } from "@vueuse/core";
+import { useStorage, refDebounced } from "@vueuse/core";
```

Comment:
If I remember correctly, vueuse utilities are auto imported. `useStorage` is special because nuxt itself has a useStorage utility.